tests/Application/.env

```diff
@@ -36,8 +36,4 @@ MAILER_URL=smtp://localhost
 MESSENGER_TRANSPORT_DSN=sync://
 ###< symfony/messenger ###
 
-###> sylius/elasticsearch-plugin ###
-BITBAG_ES_INDEX_PREFIX=
-BITBAG_ES_HOST=localhost
-BITBAG_ES_PORT=9200
-###< sylius/elasticsearch-plugin ###
+ELASTICSEARCH_URL=http://localhost:9200/
```

Comment:
What with the `BITBAG_ES_INDEX_PREFIX`? Will be without this an index collisions?

Reply:
This feature is not even documented, my code does not affect it.